com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/RoutesValidator.java

```diff
@@ -87,20 +87,21 @@ private Object attemptToCorrectParameter(ParameterValidator validator, Object pa
     @SuppressWarnings("unchecked")
     public static List<Map<String, Object>> applyRoutesType(Object routes) {
         if (routes instanceof List) {
-            List<Map<String, Object>> routesList = (List<Map<String, Object>>) routes;
+            List<Object> routesList = (List<Object>) routes;
             if (CollectionUtils.isEmpty(routesList)) {
                 return new ArrayList<>();
             }
 
             if (routesList.stream()
                           .anyMatch(route -> !(route instanceof Map))) {
-                return null;
+                return new ArrayList<>();
             }
-
-            return routesList;
+            return routesList.stream()
+                             .map(route -> (Map<String, Object>) route)
+                             .collect(Collectors.toList());
         }
 
-        return null;
+        return new ArrayList<>();
```

Comment:
Do you need a new ArrayList? Can Collections.emptyList() to be used instead?

Reply:
yes, fixed